QUARKUS-960.md

```diff
@@ -0,0 +1,29 @@
+# Quarkus-960 - Improved getting started experience with CLI
+JIRA link: https://issues.redhat.com/browse/QUARKUS-960
+
+There was an existing CLI tool in Quarkus that has been fully reworked as part of QUARKUS-960. 
+
+So, the goal is to verify proper functionality of the new Quarkus CLI tool.
+
+## Scope of the testing
+
+The Quarkus CLI is compatible with Maven, Gradle and jbang tools, however RHBQ is only focused on Maven, so Gradle and Jbang will be covered as part of the test coverage in upstream.
+
+Test development will focus on the supported features scoped for RHBQ 2.2:
+- Create applications (with a set of extensions)
+- Create extensions
+- Build existing Quarkus applications on JVM
+- Build existing Quarkus applications on Native (also using container configuration and builder images)
+- Run Quarkus applications on DEV mode
+- Run Quarkus applications on Remote Dev
```

Comment:
Do you think it's worthy to cover this feature as well? We would need to enhance the test framework to support this.

Reply:
+1 to drop it

Reply:
Deleted